client/components/aboutUs/InfoSection.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+
+import {
+  SubInfo,
+  Logo,
+  InfoBox,
+  Headline,
+  Text,
+  GreenBar,
+  HeadingGroup,
+} from "./styles";
+
+import { ABOUT_DATA, HEADER_DATA } from "./Resources";
+
+const InfoSection = () => {
+  return (
+    <>
+      <SubInfo>
+        <Logo src="/Home.png" />
+        <InfoBox>
+          <HeadingGroup>
+            <GreenBar />
+            <Headline>{HEADER_DATA.title}</Headline>
+          </HeadingGroup>
+          <Text>{HEADER_DATA.text}</Text>
```

Comment:
Header_data.text includes P-tags.
This means we will wrap P-tags in P-tags, which should be avoided :)

You could either replace this Text-element with a new component that has the styles, or use the text-elements in Header_data.text 🙂